Notebooks/historical_vote_proportions.ipynb

```diff
@@ -0,0 +1,1205 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "## Plotting proportion of votes\n",
+    "\n",
+    "totals_data = totals_df['Totals'][0], totals_df['Totals'][1]\n",
+    "\n",
+    "## Explode data\n",
+    "explode = (0.1, 0.0)\n",
+    "\n",
+    "## Color parameters\n",
+    "colors = (\"#190d2e\", \"#E22850\")\n",
```

Comment:
sorry should have been more specific:

"Approve" / "Accept" uses the "light_acceptance_green" ("#3FE1B0")
and "Reject" uses "light_rejected_red" ("#FF9AA2")

i think we have some duplicates in the cdp-design dict / that repo should be better maintained anyway

Reply:
Color change complete. Pie charts are especially hard to color which is why they are probably all but forbidden, but when I made these plots they were originally histograms and they just looked ridiculous, especially king county.